src/main/resources/db/migration/V1_1__start.sql

```diff
@@ -47,3 +47,15 @@ create table if not exists jobs(
         REFERENCES Employers(id)
         ON DELETE CASCADE
 );
+
+CREATE TABLE IF NOT EXISTS jobs_expressions_of_interest (
+    job_id      VARCHAR(36) NOT NULL,
+    prisoner_prison_number VARCHAR(7) NOT NULL,
+    created_at TIMESTAMP(6) NOT NULL,
+
+    PRIMARY KEY (job_id, prisoner_prison_number),
```

Comment:
as discussed, I will rename it to prison_number for consistency across Prison services